src/inet/IPAddress.cpp

```diff
@@ -194,7 +194,7 @@ lwip_ip_addr_type IPAddress::ToLwIPAddrType(IPAddressType typ)
 
 ip6_addr_t IPAddress::ToIPv6() const
 {
-    ip6_addr_t ipAddr = { 0 };
+    ip6_addr_t ipAddr = {};
```

Comment:
@rojer what compiler warnings are triggered by the old code?  Please include that sort of thing in commit messages that claim to fix compiler warnings; otherwise it's very hard to review the change.

Reply:
```
  CXX   /home/rojer/allterco/matter/shelly-ng/libs/shelly-matter/connectedhomeip/src/inet/IPAddress.cpp
/home/rojer/allterco/matter/shelly-ng/libs/shelly-matter/connectedhomeip/src/inet/IPAddress.cpp:197:27: error: suggest braces around initialization of subobject [-Werror,-Wmissing-braces]
    ip6_addr_t ipAddr = { 0 };
                          ^
                          {}
/home/rojer/allterco/matter/shelly-ng/libs/shelly-matter/connectedhomeip/src/inet/IPAddress.cpp:197:29: error: missing field 'zone' initializer [-Werror,-Wmissing-field-initializers]
    ip6_addr_t ipAddr = { 0 };
```

Reply:
Thank you.